store/queries/follow_tasks.sql

```diff
@@ -0,0 +1,29 @@
+-- name: EnqueueFollowTask :exec
+INSERT INTO follow_tasks (
+    actor_did,
+    next_try_at,
+    created_at,
+    should_unfollow
+) VALUES ($1, $2, $3, $4);
+
+-- name: GetNextFollowTask :one
+SELECT *
+FROM follow_tasks
+WHERE
+    next_try_at <= NOW()
+    AND finished_at IS NULL
+    AND tries < 3
+ORDER BY id ASC;
```

Comment:
Does this need a LIMIT 1 ?

Reply:
I don't think so because of the `:one` but we can add one to be defensive.

Reply:
added!